src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -362,8 +361,11 @@ const RequestDetails = ({
         <PanelTabs tabs={tabs} activeTab={activeTab} setActiveTab={setActiveTab} />
         <CloseButton buttonClass="mr-2" handleClick={closePanel} tooltip={"Close tab"} />
       </RequestDetailsTabs>
-      <div className={classNames("requestDetails", styles.requestDetails)}>
+      <div className={classNames("requestDetails relative", styles.requestDetails)}>
         <div>
+          <div className="absolute top-1 right-1">
+            <AddNetworkRequestCommentButton request={request} />
+          </div>
```

Comment:
What's the extra wrapper `<div>` for?

Reply:
No reason in particular, updated it to add the class names directly to `AddCommentButton` 😎